frontend/src/lang/en.ts

```diff
@@ -0,0 +1,14 @@
+export const en = {
+  translation: {
+    "Patient header": "COVID-19 Testing Portal",
+    "Register for your test": "Register for your test",
+    "Registration complete": "Registration complete",
+    terms: {
+      label: "Terms of service",
+      explanation: `This testing site uses <0>SimpleReport</0> to manage COVID-19 testing and reporting. The terms below explain SimpleReport's policies and terms of service.`,
```

Comment:
The `<0>SimpleReport</0>` is an example of how this library handles more complex tag replacement. The corresponding `<Trans />` component specifies a `components` array and the 0th index of that array corresponds to the component that replaces this tag. Even the docs specify that this is not super intuitive, so I can't imagine having more than a couple of these per translation string.